contrib/inventory/ec2.py

```diff
@@ -721,6 +728,51 @@ def get_elasticache_replication_groups_by_region(self, region):
         for replication_group in replication_groups:
             self.add_elasticache_replication_group(replication_group, region)
 
+    def get_autoscaling_instances_by_region(self, region):
+        ''' Makes an AWS API call to the list of Autoscale instances
+         in a particular region with tags of their Autoscale group
+         if the "propagate_at_launch" is set '''
```

Comment:
I think is a good definition of what is does, as it filters for `propagate_at_launch` set tags:
```
if tag.propagate_at_launch:
```
I wouldn't change it.